Doc/howto/logging.rst

```diff
@@ -26,9 +26,11 @@ When to use logging
 ^^^^^^^^^^^^^^^^^^^
 
 Logging provides a set of convenience functions for simple logging usage. These
-are :func:`debug`, :func:`info`, :func:`warning`, :func:`error` and
-:func:`critical`. To determine when to use logging, see the table below, which
-states, for each of a set of common tasks, the best tool to use for it.
+can be accessed by creating a logger via ``logger = getLogger(__name__)``, and
+then calling :func:`logger.debug`, :func:`logger.info`, :func:`logger.warning`,
```

Comment:
Use `: meth:` here rather than `:func:` maybe? This applies to similar usages below.

Reply:
Done